apps/website/src/components/courses/FacilitatorSwitchModal.stories.tsx

```
@@ -0,0 +1,101 @@
+      handlers: [
+        trpcStorybookMsw.facilitators.discussionsAvailable.query(async () => {
+          return new Promise(() => {
+            /* never resolves */
+          });
+        }),
+      ],
+    },
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)